scripts/exglobal_atmos_chgres_gen_control.sh

```diff
@@ -0,0 +1,122 @@
+# uncomment when the correct files are available
+cpfs "${HOMEgfs}/fix/am/global_hyblev.l${LEVS}.txt" "${DATA}/"
+cpfs "${SOURCE_DIR}/${CASE}_mosaic.nc" "${DATA}/"
+# cpfs "${COMIN_ATMOS_HISTORY_MEM}/${SFC_FILE}" "${DATA}/"
+# cpfs "${COMIN_ATMOS_HISTORY_MEM}/${ATM_FILE}" "${DATA}/"
+###############################################################################
+# copy orography,surface, and ancillary files to DATA from the source directory
+for i in {1..6}; do
+  cpfs "${SOURCE_DIR}/${CASE}_grid.tile${i}.nc" "${DATA}/"
+  cpfs "${SOURCE_DIR}/${CASE}.mx${OCNRES}_oro_data.tile${i}.nc" "${DATA}/"
+  cpfs "${SOURCE_DIR}/sfc/${CASE}.mx${OCNRES}.slope_type.tile${i}.nc" "${DATA}/"
+  cpfs "${SOURCE_DIR}/sfc/${CASE}.mx${OCNRES}.maximum_snow_albedo.tile${i}.nc" "${DATA}/"
+  cpfs "${SOURCE_DIR}/sfc/${CASE}.mx${OCNRES}.snowfree_albedo.tile${i}.nc" "${DATA}/"
+  cpfs "${SOURCE_DIR}/sfc/${CASE}.mx${OCNRES}.soil_type.tile${i}.nc" "${DATA}/"
+  cpfs "${SOURCE_DIR}/sfc/${CASE}.mx${OCNRES}.vegetation_type.tile${i}.nc" "${DATA}/"
+  cpfs "${SOURCE_DIR}/sfc/${CASE}.mx${OCNRES}.substrate_temperature.tile${i}.nc" "${DATA}/"
+  cpfs "${SOURCE_DIR}/sfc/${CASE}.mx${OCNRES}.vegetation_greenness.tile${i}.nc" "${DATA}/"
+  cpfs "${SOURCE_DIR}/sfc/${CASE}.mx${OCNRES}.facsf.tile${i}.nc" "${DATA}/"
+done
+################################################################################
+# add the namelist and run chgres
+cat << EOF > ./fort.41
+&config
+mosaic_file_target_grid="./${CASE}_mosaic.nc"
+fix_dir_target_grid="./"
+orog_dir_target_grid="./"
+orog_files_target_grid="${CASE}.mx${OCNRES}_oro_data.tile1.nc","${CASE}.mx${OCNRES}_oro_data.tile2.nc","${CASE}.mx${OCNRES}_oro_data.tile3.nc","${CASE}.mx${OCNRES}_oro_data.tile4.nc","${CASE}.mx${OCNRES}_oro_data.tile5.nc","${CASE}.mx${OCNRES}_oro_data.tile6.nc"
+vcoord_file_target_grid="./global_hyblev.l${LEVS}.txt"
+mosaic_file_input_grid="NULL"
+orog_dir_input_grid="NULL"
+orog_files_input_grid="NULL"
+data_dir_input_grid="./"
+atm_files_input_grid="${ATM_FILE}"
+atm_core_files_input_grid="NULL"
+atm_tracer_files_input_grid="NULL"
+sfc_files_input_grid="${SFC_FILE}"
+nst_files_input_grid="NULL"
+grib2_file_input_grid="NULL"
+geogrid_file_input_grid="NULL"
+varmap_file="NULL"
+wam_parm_file="NULL"
+cycle_year=${iy}
+cycle_mon=${im}
+cycle_day=${id}
+cycle_hour=${cyc}
+convert_atm=.true.
+convert_sfc=.true.
+convert_nst=.true.
+input_type="gaussian_netcdf"
+tracers="sphum","liq_wat","o3mr","ice_wat","rainwat","snowwat","graupel"
+tracers_input="spfh","clwmr","o3mr","icmr","rwmr","snmr","grle"
+regional=0
+halo_bndy=0
+halo_blend=0
+sotyp_from_climo=.true.
+vgtyp_from_climo=.true.
+vgfrc_from_climo=.true.
+minmax_vgfrc_from_climo=.true.
+tg3_from_soil=.false.
+lai_from_climo=true.
+external_model="GFS"
+nsoill_out=4
+thomp_mp_climo_file="NULL"
+wam_cold_start=.false.
+/
+EOF
+
+eval "${APRUN_CHGRES}" "${CHGRESEXEC}"
+################################################################################
```

Comment:
```suggestion
${APRUN_CHGRES} "${CHGRESEXEC}"
export err=$?
if [[ ${err} -ne 0 ]]; then
  err_exit "chgres_cube failed to create cold start ICs, ABORT!"
fi
```
APRUN_CHGRES should not be " ", it will give error.
Trap the error, if any after running executable.